pallets/phala/src/compute/basepool.rs

```diff
@@ -317,6 +344,9 @@ pub mod pallet {
 		T: pallet_assets::Config<AssetId = u32, Balance = BalanceOf<T>>,
 		T: Config + pawnshop::Config + vault::Config,
 	{
+		/// Returns a nft_guard object that can read or write nft-attr of a nft
```

Comment:
```suggestion
		/// Returns a [`NftGuard`] object that can read or write to the nft attributes
```

If you refer to a type, write the precise type name quoted like this: `[nft_guard]`

Reply:
don e